### ISSUE TITLE
fix(pagination): Unable to use Pagination

### ISSUE THREAD

**Issue by louisscruz** [valor-software/ngx-bootstrap#59]
I can't get Pagination to work with the following. Please forgive my lack of Angular 2 understanding.

```
import {Component, Injectable} from 'angular2/core';
import {Http, Headers} from 'angular2/http';
import {CORE_DIRECTIVES, FORM_DIRECTIVES} from 'angular2/common';
import {PAGINATION_DIRECTIVES} from 'ng2-bootstrap/ng2-bootstrap';

import {PostService} from '../services/posts/postsService';
import {Post} from '../datatypes/post/post';

@Component({
       selector: 'posts',
       template: require('./posts.html'),
       directives: [PAGINATION_DIRECTIVES],
       bindings: [PostService]
})

 export class Posts {
    private posts: Array<Post>;
    private totalItems: number = 64;
    private currentPage: number = 1;
    private pageChanged(event: any):void {
         console.log(event);
    };
   constructor(public postService: PostService) {
       postService.getPosts()
       .subscribe(
            res => {
            this.posts = res;
       },
      err => console.log(err),
      () => console.log('retrieved posts')
      );
   }
 }
```

And the HTML:

```
<div class="col-xs-12 col-xs-offset-0 col-md-8 col-md-offset-2">
  <table>
    <tbody>
       <tr *ngFor="#post of posts; #i=index">
         <th>{{i + 1}}.</th>
         <td><a href="{{post.url}}" target="_blank">{{post.title}}</a></td>
         <td><small>comments</small></td>
       </tr>
     </tbody>
   </table>
  {{currentPage}}
   <pagination [totalItems]="totalItems" [(ngModel)]="currentPage" (page-changed)="pageChanged($event)"></pagination>
 </div>
```

**Comment by valorkin** [comment]
Hey, can you create pencil or plunker with your code please?
any particular issues?

**Comment by louisscruz** [comment]
Taking a quick look at this again, it seems that the rendered classes don't match those specified by Bootstrap 4. The classes used in the [PAGINATION_TEMPLATE](https://github.com/valor-software/ng2-bootstrap/blob/master/components/pagination/pagination.ts) make use of  `class="pagination-page"`. [Bootstrap 4 requires "page-item" and "page-link" classes.](http://v4-alpha.getbootstrap.com/components/pagination/)

**Comment by louisscruz** [comment]
I just made a pull request adding the appropriate classes. Let me know if there's some other way to handle this.

**Comment by valorkin** [comment]
I will test and land it today to npm

**Comment by valorkin** [comment]
published as 1.0.0-beta.2
plus some tweaks, you will like :)
